### PR TITLE
dharun016 - Cyber Drone

### PR DESCRIPTION
**Pull request by dharun016**
## Describe your project in 1-3 sentences

 Nowadays Quadcopters are playing major roles in all fields . So, it is exciting to do some projects on it. I do so much research on that finally I came up with idea that connecting Raspberry Pi to the quadcopter to create scripts that will allow us to prevent attacks and gain access

## Hack Club Slack handle
dharun_t3

## Checklist before submitting proposal:

- [x] I am a current high school student (or younger).
- [x] I have submitted an RSVP at <https://hackclub.com/winter>
- [x] I am a member of the Hack Club Slack (Join at <https://hackclub.com/slack>)
- [x] I have self-reviewed my project proposal.

### CODE REVIEW

File: dharun016.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+---
+name: "@dharun016"
+project: "Cyber Drone"
+---
+
+# Hack Club Bank Sign
```
Comment (bellesea) [inline dharun016.md:6]:
```suggestion
```

File: dharun016.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+![Hack Club Bank logo illuminated from behind with coloured LEDs](https://cloud-hysdwonkf-hack-club-bot.vercel.app/0img_2814.jpg)
```
Comment (bellesea) [inline dharun016.md:8]:
```suggestion
```

**Review by bellesea — CHANGES_REQUESTED** [review]
oh wow @dharun016 this sounds awesome!! what do you mean by "defence wireless attacks and take over control"? what type of attacks are you talking about and what's the role of a drone in defending against it?

also, how do you plan on integrating the raspberry pi w/ the drone?

thank you so much!

**Comment by dharun016** [comment]
In malls, railways, and other public places some hackers provide free wifi's connections. Some people don't know about this so they use wifi as normal users. But connecting **hackers** will gain full to their devices and use it for personal gain then they stole their data and blackmail them for money So, there are so many peoples affected by it. 

So, I have a plan of connecting with Raspberry Pi which is Linux based OS it is easy to write scripts, I use tools like **Nmap, aircrack-ng, and others tools** to combine all of them to first scan all the wifi connections with Nmap (**hackers mostly run their connection on port 80 for gain access**) then collect that connection and analyze it with aircrack-ng to drop down the connection and take control their access with twin evil attack finally we fill the complaint with the with data(**IP, location  etc**)we have,

**Comment by bellesea** [comment]
> 

oh wow! that sounds cool! so your plan is to fly the drone w/ the pi to scan these areas?

**Comment by dharun016** [comment]
Yep, scanning phishing wifi connection a

**Comment by bellesea** [comment]
> Yep, scanning phishing wifi connection a

great! one super small thing, could you please recommit the exu3.md file? after that, we should be good to move onto next steps!!

File: exu3.md [hunk 3]
```diff
@@ -1,27 +0,0 @@
----
```
Comment (dharun016) [inline exu3.md:1]: recommit

**Comment by dharun016** [comment]
Done

**Review by bellesea — APPROVED** [review]
wahoo!!! this is AWESOME @dharun016 :) an awesome member of @hackclub/bank-ops will be reaching out w/ next steps soon!

**Comment by hackliv** [comment]
Hey @dharun016 this drone sounds cool! Looking at the file, I was a bit confused why "Hack Club Bank Sign" was the header and image. Is this a second project you're hoping to work on? Let me know you Slack handle too!

**Comment by bellesea** [comment]
hey @dharun016 could you please update your PR w/ the information you shared in this comments w/ regards to your plan for your drone? 

please let us know if you need any help with this, thank you so much!

**Review by bellesea — CHANGES_REQUESTED** [review]
hey @dharun016 could you please update your PR w/ the information you shared in this comments w/ regards to your plan for your drone?

please let us know if you need any help with this, thank you so much!

**Comment by dharun016** [comment]
I add all comments to my readme file Thank you for help

**Review by bellesea — APPROVED** [review]
thank you so much!!

**Comment by dharun016** [comment]
Is there any procedures?

**Comment by bellesea** [comment]
> Is there anyother procedures?

@dharun016 it looks good! thanks for following up :) we’re on a small crew right now due to the holidays but a cool peep from @hackclub/bank-ops will be following up with next steps!

**Review by MelSmith104 — COMMENTED** [review]
hi @dharun016 ! this project sounds very interesting and i'd love to learn more! @hackliv is currently out of office for the holidays so i'll be the one dm'ing you to chat about next steps!

**Comment by dharun016** [comment]
I Don't Eligible For This Program

**Pull request closed without merging**